src/backend/wiki_game_info/wiki_game_info.ts

```diff
@@ -48,12 +49,16 @@ export async function getWikiGameInfo(
         isMac ? getInfoFromAppleGamingWiki(title) : null
       ])
 
+    const protondb = await getInfoFromProtonDB(
```

Comment:
Don't call this with empty string better Just call it if steamID is not empty.

Reply:
What should it be set as if SteamID is empty?

Reply:
First i thought about just not call `getInfoFromProtonDB` if `steamID` is empty, but that will not print any warning or error message that the information could not be fetched because of missing `steamID`. Easiest thing would be to allow passing `string | undefined` instead `string`:

```typescript
// protondb/utils.ts
export async function getInfoFromProtonDB(
  steamID: string | undefined
): Promise<ProtonDBCompatibilityInfo | null> {
  if !steamID {
    logDebug('No SteamID, not getting ProtonDB info')
    return null
  }
  // ...
 }
 
// wiki_game_info.ts

// ...
const protondb = await getInfoFromProtonDB(gamesdb.steamID ?? pcgamingwiki.steamID)
// ...
```